docs/web/errors.mdx

```diff
@@ -170,6 +197,33 @@ try {
 }
 ```
 
+## Timeouts
+
+Similar to the `signal` option for cancellation, there is also the `timeoutMs` option, which
+allows you to specify a timeout, in milliseconds, for an individual call in instances
+where the request is hung up or not responding. The timeout value is specified in the same
+options object described above.
+
+When a timeout is reached before the request has been completed, a `ConnectError` with code `DeadlineExceeded` will
+be thrown.
+
+An example using a timeout of 3 seconds:
+
+```ts
+try {
+  await client.say({sentence: "Hello"}, { timeoutMs: 3000 });
+} catch (err) {
+  if (err instanceof ConnectError && err.code === Code.DeadlineExceeded) {
+    // handle the timeout error
+  }
+}
+```
+
+Timeouts in Connect are the equivalent of [deadlines](https://grpc.io/docs/guides/deadlines/) in gRPC. They both
+behave similarly in that they will throw a "deadline exceeded" error. Since Connect's defined error codes map to the
+same codes and semantics as gRPC, this allows for easy interop between the two protocols.
+
+
 There is no silver bullet to error handling with async/await and cancellation.
```

Comment:
This paragraph refers to cancellation, so it's out of place now at the bottom of the section on timeouts.

The paragraph reflects on the issue that you may want to cancel calls in regular flow, but abort signals force you to handle exceptional flow. The concept of abort signals is at least heavily inspired from C#, but we're lacking conditional catch clauses in ES for good DX. 

Depending on how you use them, handling Connect timeouts coerce you to do the same. 

Can you try to adapt so that it fits both? If it does not work out, let's remove the paragraph.